src/en/lynxscans/build.gradle

```
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'LynxScans'
+    pkgNameSuffix = 'en.lynxscans'
+    extClass = '.LynxScans'
+    extVersionCode = 1
```

```suggestion
    extVersionCode = 7
```

Bump this version to let users get update.

---

will users get an update if the extension id changes anyway?

---

No. Update check is based on extension package name and version.